cmd/daprd/options/options_test.go

```diff
@@ -21,112 +21,304 @@ import (
-	opts := New([]string{
+	opts, err := New([]string{
 		"--app-id", "testapp",
 		"-mode", string(modes.StandaloneMode), // Single dash
 		"--config", "../../../pkg/config/testdata/metric_disabled.yaml",
 		"--metrics-port", strconv.Itoa(10000),
 	})
+	require.NoError(t, err)
 	assert.EqualValues(t, "testapp", opts.AppID)
 	assert.EqualValues(t, string(modes.StandaloneMode), opts.Mode)
 	assert.Equal(t, []string{"../../../pkg/config/testdata/metric_disabled.yaml"}, opts.Config)
 }
 
 func TestEnableAPILogging(t *testing.T) {
 	t.Run("explicitly enabled", func(t *testing.T) {
-		opts := New([]string{
+		opts, err := New([]string{
 			"-enable-api-logging", // Single dash
 		})
+		require.NoError(t, err)
 		require.NotNil(t, opts.EnableAPILogging)
 		assert.True(t, *opts.EnableAPILogging)
 	})
 	t.Run("explicitly enabled with true written out", func(t *testing.T) {
-		opts := New([]string{
+		opts, err := New([]string{
 			"--enable-api-logging=true",
 		})
+		require.NoError(t, err)
 		require.NotNil(t, opts.EnableAPILogging)
 		assert.True(t, *opts.EnableAPILogging)
 	})
 
 	t.Run("explicitly disabled", func(t *testing.T) {
-		opts := New([]string{
+		opts, err := New([]string{
 			"-enable-api-logging=false", // Single dash
 		})
+		require.NoError(t, err)
 		require.NotNil(t, opts.EnableAPILogging)
 		assert.False(t, *opts.EnableAPILogging)
 	})
 
 	t.Run("flag is unset", func(t *testing.T) {
-		opts := New([]string{})
+		opts, err := New([]string{})
+		require.NoError(t, err)
 		require.Nil(t, opts.EnableAPILogging)
 	})
 }
 
 func TestMultipleConfig(t *testing.T) {
 	t.Run("config flag not defined", func(t *testing.T) {
-		opts := New([]string{})
+		opts, err := New([]string{})
+		require.NoError(t, err)
 		require.Empty(t, opts.Config)
 	})
 
 	t.Run("single config", func(t *testing.T) {
-		opts := New([]string{
+		opts, err := New([]string{
 			"--config", "cfg1.yaml",
 		})
+		require.NoError(t, err)
 		require.Equal(t, []string{"cfg1.yaml"}, opts.Config)
 	})
 
 	t.Run("comma-separated configs", func(t *testing.T) {
-		opts := New([]string{
+		opts, err := New([]string{
 			"-config=cfg1.yaml,cfg2.yaml", // Single dash
 		})
+		require.NoError(t, err)
 		require.Equal(t, []string{"cfg1.yaml", "cfg2.yaml"}, opts.Config)
 	})
 
 	t.Run("multiple config flags", func(t *testing.T) {
-		opts := New([]string{
+		opts, err := New([]string{
 			"-config=cfg1.yaml",    // Single dash
 			"-config", "cfg2.yaml", // Single dash
 		})
+		require.NoError(t, err)
 		require.Equal(t, []string{"cfg1.yaml", "cfg2.yaml"}, opts.Config)
 	})
 
 	t.Run("multiple config flags and comma-separated values", func(t *testing.T) {
-		opts := New([]string{
+		opts, err := New([]string{
 			"-config=cfg1.yaml", // Single dash
 			"--config", "cfg2.yaml,cfg3.yaml",
 		})
+		require.NoError(t, err)
 		require.Equal(t, []string{"cfg1.yaml", "cfg2.yaml", "cfg3.yaml"}, opts.Config)
 	})
 }
 
+func TestMaxBodySize(t *testing.T) {
```

Comment:
Please add upward out of bounds error.

Reply:
Since our GH Actions runner are 64-bit, it is not possible to write a test like that, because the constant `buildinfo.PtrSize` cannot be 32